frontend/app/model/FeatureChoice.scala

```diff
@@ -30,6 +32,8 @@ case object Books extends FeatureChoice {
   override val label = "4 free books"
 }
 case object FreeEventTickets extends FeatureChoice {
+  val allowance = zuoraFreeEventTicketsAllowance
   override val zuoraCode = "Events"
-  override val label = "6 free events"
+  override val label = s"$allowance free events"
+  val uom = "Events"
```

Comment:
Is it worthwhile putting a comment or renaming the val to smth like `unitName` or `unitsToMeasure`?